hibikase/PlaybackWidget.h

```diff
@@ -39,6 +39,7 @@ public slots:
     void Stop();
 
 signals:
+    void LoadResult(QString error);
```

Comment:
This isn't how you typically would name a signal. I'd suggest a name like "Loaded" (or "AudioLoaded"?) instead.

Reply:
I want to convey that it could also return an error

Reply:
Hmm... Maybe LoadFinished or InitializationFinished? Does that sound less like it always succeeds?